dependencies/trusty/kafo/control

```diff
@@ -2,20 +2,25 @@ Source: ruby-kafo
 Section: ruby
 Priority: optional
 Maintainer: Greg Sutcliffe <gsutclif@redhat.com>
-Build-Depends: debhelper (>= 7.0.50~), gem2deb (>= 0.3.0~)
 Standards-Version: 3.9.3
+Build-Depends: debhelper (>= 7.0.50~),
+               gem2deb
 Homepage: https://github.com/theforeman/kafo
 XS-Ruby-Versions: all
 
 Package: ruby-kafo
 Architecture: all
 XB-Ruby-Versions: ${ruby:Versions}
-Depends: ${shlibs:Depends}, ${misc:Depends}, ruby | ruby-interpreter,
-         puppet (< 4.0.0),
-         ruby-clamp,
-         ruby-highline (< 1.7),
+Depends: ruby | ruby-interpreter,
+         puppet (<< 4.0.0),
+         ruby-clamp (>= 0.6.2),
+         ruby-highline (<< 1.7),
+         ruby-highline (>= 1.6.21),
```

Comment:
This has caused a problem on trusty, as it ships .20: http://packages.ubuntu.com/trusty/ruby-highline.  We build our own for wheezy and precise, while jessie has .21.

http://ci.theforeman.org/job/systest_foreman/7272/console